OptionSelector/DetailCategoryTableViewController.m

```diff
@@ -0,0 +1,55 @@
+//
+//  DetailCategoryTableViewController.m
+//  OptionSelector
+//
+//  Created by Jason Wang on 8/15/15.
+//  Copyright (c) 2015 Mike Kavouras. All rights reserved.
+//
+
+#import "DetailCategoryTableViewController.h"
+
+@interface DetailCategoryTableViewController ()
+
+@end
+
+@implementation DetailCategoryTableViewController
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+
+}
+
+
+#pragma mark - Table view data source
+
+- (NSInteger)numberOfSectionsInTableView:(UITableView *)tableView {
+    return 1;
+}
+
+- (NSInteger)tableView:(UITableView *)tableView numberOfRowsInSection:(NSInteger)section {
+    return self.category.options.count;
+}
+
+- (UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(NSIndexPath *)indexPath {
+    UITableViewCell *cell = [tableView dequeueReusableCellWithIdentifier:@"DetailViewIdentifier" forIndexPath:indexPath];
+    
+    NSString *categorySelected = self.category.options[indexPath.row];
+    cell.textLabel.text = categorySelected;
```

Comment:
This will cause your cells' text labels to be blank before an option is selected in the detail view. If you choose a different cell style in Interface Builder (e.g. "Right Detail"), then you can set the value of `textLabel.text` to the category name, and set `detailTextLabel.text` to the selection.